matchbox/models/models.py

```diff
@@ -125,11 +125,13 @@ def get_fields(self):
             for f in self._meta.fields.values()
         }
 
-    def save(self, update_fields=None):
+    def save(self, update_fields=None, req_db=None):
         if update_fields is not None:
+            if req_db is not None:
```

Comment:
this is not consistent because you create object using req_db and after that you can't update it

Reply:
I think, the better way to add support for multiple db's will be way like flask supported this (https://flask-sqlalchemy.palletsprojects.com/en/2.x/binds/)

Reply:
I am sure there is a better way to handle this but I needed a quick way to move documents from a collection to another collection (as there is no easy way to do this). Again, I don't insist on this.